collections/_documentation/next-steps.md

```diff
@@ -114,22 +111,25 @@ app.use(router.get('/article/:id', async ctx => {
 // engine-specific code here
 ```
 
+{:.note.hint}
+> On **lambda** or **bun** use **`@curveball/kernel`** instead of `@curveball/core`
+
 By default Curveball will respond with `200 OK` _if_ a response body was set.
 It also defaults the `Content-Type` to `application/json`.
 
-Note that we made the middleware async. _all_ Curveball middlewares may be
+Note that we made the middleware `async`. **_All_** Curveball middlewares may be
 async.
 
 A `PUT` request on the same route
 ----------------------------------
 
-Lets assume we also want to do a `PUT` request here to let people update
+Let's assume we also want to do a `PUT` request here to let people update
 articles.
 
 
 ```typescript
 import router from '@curveball/router';
-import { Application } from '@curveball/core'; // on lambda or bun this is 'kernel'
```

Comment:
Not a strong opinion on this, but I figured it would be nice to call it out in a info/hint blockquote, could also make it a small note later in the doc (ie, not adding the `>` blockquote in recurring instances)?